synapse/config/registration.py

```diff
@@ -98,6 +99,9 @@ def default_config(self, **kwargs):
         # to these rooms
         #auto_join_rooms:
         #    - "#example:example.com"
+
+        # Have first user on server autocreate autojoin rooms
```

Comment:
maybe explain what will happen if it is false, and conversely what happens if it is true?